resources/public/api/conf/5.0/foreign_property_cumulative_summary_retrieve.yaml

```diff
@@ -0,0 +1,99 @@
+description: |
+  This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.
+
+  Note: This endpoint only supports submissions for tax year 2025-26 onwards.
```

Comment:
There are two instances where the 2025-26 rule is mentioned: `Please note that this endpoint is only available for tax years starting from 2025-26.` and  `Note: This endpoint only supports submissions for tax year 2025-26 onwards.` The current specification uses this version: `Please note that this endpoint is only available for tax years starting from 2025-26.`, so maybe we should stick with this wording.

Also, please wrap the text with strong tags for emphasis and semantic semantic meaning, like this: `<strong>Please note that this endpoint is only available for tax years starting from 2025-26</strong>`.

This updated text will also need to be added to the create amend endpoint, as it is missing there. Thanks!